### ISSUE TITLE
Errors if a different version of jQuery is already loaded

### ISSUE THREAD

**Issue by pmontrasio** [infinum/cookies_eu#19]
Some jquery plugins in my pages stopped working after I added cookies_eu. I eventually realized that the first line in ```cookies_eu.js``` was loading jquery (v 1.11.2 from its jquery-rails dependency). However I already loaded jquery v 2.0.2 and the plugins don't seem to like it. I tried to moved cookies_eu at the top of application.js to no avail or switching to ```//= require jquery2```. I eventually commented out ```//= require jquery``` and everything worked, but this was unsatisfactory.

I had to include directly jquery directly in ```application.html.erb``` to be able to do something like

```
<!--[if (!IE)|(gt IE 8)]><!-->
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.0.2/jquery.min.js"></script>
<!--<![endif]-->

<!--[if lte IE 8]>
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
<![endif]-->
```

So another solution was to remove those lines and add ```//= require jquery2``` in your gem and at the top of application.js. That worked too but I'm losing support for IE8, which is required in this project.

Furthermore with both solutions I'll have to fork your gem to be able to deploy it.

If I have to fork it I'll go for the first solution and remove the dependency from jquery-rails. Could you suggest a better solution? Thanks.

**Comment by alepore** [comment]
so removing all dependencies on jquery and just saying "you need a working copy of jquery" is the best solution?
:+1: for me

**Comment by d4be4st** [comment]
fixed with #15
